### PR TITLE
Enable Cluster Serving message encrytpion

### PR DESCRIPTION
**Pull request by qiyuangong**

### CODE REVIEW

File: zoo/src/main/scala/com/intel/analytics/zoo/serving/preprocessing/PreProcessing.scala [hunk 1]
```diff
@@ -26,6 +26,7 @@ import org.apache.log4j.Logger
 
 import scala.collection.mutable.ArrayBuffer
 import com.intel.analytics.bigdl.utils.{T, Table}
+import com.intel.analytics.zoo.serving.http.FrontEndApp.decryptWithAES256
```
Comment (glorysdj) [inline zoo/src/main/scala/com/intel/analytics/zoo/serving/preprocessing/PreProcessing.scala:29]: EncryptSupportive is a trait, PreProcessing can extends it, and call encryptWithAES256/decryptWithAES256 directly

File: zoo/src/main/scala/com/intel/analytics/zoo/serving/utils/ClusterServingHelper.scala [hunk 2]
```diff
@@ -80,6 +80,7 @@ class ClusterServingHelper(_configPath: String = "config.yaml", _modelDir: Strin
   var redisSecureTrustStoreToken: String = null
 
   var modelEncrypted: Boolean = false
+  var recordEncrypted: Boolean = false
```
Comment (glorysdj) [inline zoo/src/main/scala/com/intel/analytics/zoo/serving/utils/ClusterServingHelper.scala:83]: we really should define a class of the config yaml, and parse the object directly, not add a field then add a method...

File: zoo/src/main/scala/com/intel/analytics/zoo/serving/preprocessing/PreProcessing.scala (hunk 1 above)
Comment (qiyuangong) [reply zoo/src/main/scala/com/intel/analytics/zoo/serving/preprocessing/PreProcessing.scala]: Already addressed. Thanks

**Review by glorysdj — APPROVED** [review]
LGTM

**Comment by qiyuangong** [comment]
Jenkins passed http://10.239.47.210:18888/job/ZOO-PR-Validation/5196, excepted a TF related fail, which is not related to this PR,

**Pull request merged**